include/boost/basic_any.hpp

```diff
@@ -0,0 +1,440 @@
+// See http://www.boost.org/libs/any for Documentation.
+
+#ifndef BOOST_BASIC_ANY_INCLUDED
+#define BOOST_BASIC_ANY_INCLUDED
+
+#if defined(_MSC_VER)
+# pragma once
+#endif
+
+// what:  variant type boost::any
+// who:   contributed by Kevlin Henney,
+//        with features contributed and bugs found by
+//        Antony Polukhin, Ed Brey, Mark Rodgers, 
+//        Peter Dimov, and James Curran
+// when:  July 2001, April 2013 - 2019
+
+#include <algorithm>
```

Comment:
Let's concentrate on C++11 implementation. In that case we can use a more lightweight `<utility>` header for getting the `swap` and other helpers.

Reply:
As we discussed for potential boost::any replacement let's leave composability with C++98. If you want me to add preprocessor branch to include utility for C++11 just let me know